components/Cards/card.css

```diff
@@ -277,129 +199,109 @@ span{
     text-decoration: line-through;
 }
 
-/* Media   Queries */
-@media only screen and (max-width: 710px) {
-    .horizontal-card{
-        gap: 1em;
-    }
-    .horizontal-card .card-header img{
-        max-height: 25em;
-        height: 100%;
-    }
-    .horizontal-card .card-header{
-        max-height: 20em;
-        max-width: 20em;
-    }
-    .horizontal-card .card-content .title{
-        font-size: 1.35em;
-    }
-
-    .horizontal-card .card-content .rating{
-        font-size: 1.25em;
-    }
-    .horizontal-card .card-description .price{
-        font-size: 1.75em;
-    }
-    .horizontal-card .card-content .card-description .discount-txt{
-        font-size: 0.65em;
-    }
-    .horizontal-card .card-description p{
-        font-size: 1.25em;
-    }
-
-    .horizontal-card .card-footer .btn{
-        font-size: 1.25em;
-        height: 2.5em;
-        max-width: 8em;
-        padding-left: 1em;
-        padding-right: 1em;
-    }
-    .horizontal-card .card-footer .footer-1 > p,
-    .horizontal-card .card-footer > p{
-        font-size: 1.25em;
-    }
-    .horizontal-card .card-footer .footer-1 p span{
-        font-size: 1em;
-    }
-    .horizontal-card .card-footer .footer-1 > img{
-        max-width:5em;
-    }
+/* PRODUCT */
+.product-card-container{
+    width: 30rem !important;
+    padding-inline: 1.5rem;
+}
+.product-card{
+    background-color: var(--clr-white);
+    border-radius: 0.5rem;
+    box-shadow: var(--box-shadow-clr); 
+    flex-direction: column;
+    height: max-content;
+    margin:0.5rem 0;
+    padding-bottom:1.5rem;
+    max-inline-size: 100%;
+    overflow: hidden;
+    min-width: 20%;
+}
+.product-header{
+    overflow: hidden;
+}
+/* WISHLIST-BADGE */
+.wishlist-badge{
+    width: max-content;
+    position: absolute;
+    opacity: 1;
+    right: 1rem;
+    top: 1rem;
+    z-index: 2;
+    background-color: var(--clr-white);
+    border-radius: 50%;
+    padding: .5rem;
+}
+.wishlist-badge .icon:hover{
+    transform: scale(1.1);
+    color: var(--primary-clr);
+}
+.wishlist-badge .icon{
+    height: 2.1rem;
+}
+/* PRODUCT IMAGE */
+.product-card .product-header .product-img-container{
+    padding-top: 125%;
+}
+.product-card .product-header .product-img-container > img{
+    width: 100%;
+    object-fit: cover;
+    max-inline-size: 100%;
+    display: block;
+    position: absolute;
+    top: 0;
+}
+.product-card .btn-container{
+    padding-inline: 2rem;   
+}
+/* || HOVER CART BTN */
+.hover-cart-btn-container{
+    display: none;
 }
 
-@media only screen and (max-width:560px){
-    .horizontal-card{
-        gap: 0;
-    }
-    .horizontal-card .card-header img{
-        max-height: 20em;
-        height: 100%;
-    }
-    .horizontal-card .card-header{
-        max-height: 15em;
-        max-width: 15em;
-    }
+.product-card .product-content{
+    padding:10px 10px 5px 10px;
+    background-color: var(--clr-white);
+    max-height: 10rem;
+    overflow: hidden;
```

Comment:
Increase the `max-height` by 0.5rem or 1 rem. The product price is cutting off a bit on full screen

Reply:
added padding